integration/helpers/base_test.py

```diff
@@ -560,6 +560,17 @@ def verify_options_request(self, url, expected_status_code, headers=None):
             )
         return response
 
+    def verify_post_request(self, url: str, body_obj, expected_status_code: int):
+        """Return response to POST request and verify matches expected status code."""
+        response = self.do_post_request(url, body_obj)
+        if response.status_code != expected_status_code:
+            raise StatusCodeError(
+                "Request to {} failed with status: {}, expected status: {}".format(
+                    url, response.status_code, expected_status_code
+                )
```

Comment:
nit: we should switch to use f-string now like we did for new code. Fine for testing code